battray/platforms.py

```diff
@@ -40,7 +40,7 @@ def freebsd():
 
 	o = subprocess.Popen(['acpiconf', '-i0'], stdout=subprocess.PIPE).communicate()[0].decode()
 
-	for line in o.split('\n'):
+	for line in o.decode().split('\n'):
```

Comment:
There is already a `decode()` a few lines above this; adding it here again seems strange?